src/utils/getConnectedIssueForPR.js

```diff
@@ -0,0 +1,29 @@
+const targetRegexes = [
+  /*
+   * these are strings to stop the linter from complaining about the named capture groups
+   * which is an error that cannot be suppressed
+   **/
+  /* eslint-disable-next-line */
+  '^(.* )?link: (?<owner>openstax)\/(?<repo>[a-z\-]+)#(?<number>[0-9]+)( .*)?$',
```

Comment:
Why is the text `link: ` necessary? If it is to differentiate from someone saying something like `This is a followup to #123` then maybe additional phrases should be included:

- `refs #123`
- `fixes openstax/foo#123`

Reply:
yea i agree that the word "link" could be better. maybe just `for: openstax/foo#123` would be clearer